wurst/network/Network.wurst

```diff
@@ -0,0 +1,922 @@
+package Network
+
+import SyncSimple
+import Table
+import ErrorHandling
+import MoriUtil
+
+// max amount to be written/read/synced per an execution to avoid hitting the OP limit
+// this value can be higher if Wurst is compiled without stack traces and with all optimizations turned on
+// and even higher with safety checks off
+@configurable constant integer DATA_PER_EXECUTE = 256
```

Comment:
you should omit the type of constants. perhaps also consider replacing all "integer" with "int"

Reply:
Haha, yeah, thanks!